frontend/src/components/atoms/constants/ThemeMUI.tsx

```diff
@@ -1,15 +1,15 @@
-import {createMuiTheme} from "@material-ui/core";
-import {grey} from "@material-ui/core/colors";
+import { unstable_createMuiStrictModeTheme as createMuiTheme } from "@material-ui/core";
```

Comment:
Cikawe, co to robi i dlaczego zmieniles :) ?

Reply:
https://stackoverflow.com/questions/61220424/material-ui-drawer-finddomnode-is-deprecated-in-strictmode
ma miałem warning w konsoli i albo to go rozwiązuje, albo usunięcie tagów StrictMode. Wybrałem mniejsze zło, chyba :-P